Common/src/gRPC/Services/Submitter.cs

```diff
@@ -518,16 +518,6 @@ await foreach (var taskRequest in taskRequests.WithCancellation(cancellationToke
       await taskTable_.SetTaskSuccessAsync(taskDataEnd,
                                            cancellationToken)
                       .ConfigureAwait(false);
-
-
-      //Discard value is used to remove warnings CS4014 !!
-      _ = Task.Factory.StartNew(async () => await objectStorage_.TryDeleteAsync(taskData.TaskId,
-                                                                                CancellationToken.None)
-                                                                .ConfigureAwait(false),
-                                cancellationToken);
-
-      logger_.LogInformation("Remove input payload of {task}",
-                             taskData.TaskId);
```

Comment:
You should keep this for now. It enables early deletion of payloads during the session processing.